Unable to export Explorations

## Description
<!-- A clear and concise description of what the bug is. -->
When using the Explorations feature and attempting to export it to CSV, an error occurs during the download. Going directly to <instanceurl>/api/export/v0/explorations/?database_id=<...>&exploration_id=<...> shows a "Internal Server Error" page. The error that appears within the container's logs (with debug mode on) is:

```shell
mathesar_service  | [2026-05-05 23:48:40 +0000] [41] [DEBUG] GET /api/export/v0/explorations/
mathesar_service  | [2026-05-05 23:48:41 +0000] [41] [ERROR] Error handling request /api/export/v0/explorations/?database_id=14&exploration_id=6
mathesar_service  | Traceback (most recent call last):
mathesar_service  |   File "/usr/local/lib/python3.14/site-packages/gunicorn/workers/sync.py", line 134, in handle
mathesar_service  |     self.handle_request(listener, req, client, addr)
mathesar_service  |     ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
mathesar_service  |   File "/usr/local/lib/python3.14/site-packages/gunicorn/workers/sync.py", line 182, in handle_request
mathesar_service  |     for item in respiter:
mathesar_service  |                 ^^^^^^^^
mathesar_service  |   File "/code/mathesar/views/export.py", line 38, in export_exploration_csv_in_chunks
mathesar_service  |     columns = next(exploration_chunk_gen)
mathesar_service  |   File "/code/mathesar/utils/explorations.py", line 198, in exploration_chunker
mathesar_service  |     exp_model = get_exploration(exploration_id)
mathesar_service  | TypeError: get_exploration() missing 1 required positional argument: 'database_id'
```

It appears to be related to these lines missing an argument: 

https://github.com/mathesar-foundation/mathesar/blob/a5466472b1dcf10ca0ed06999d555b1db593ae56/mathesar/views/export.py#L37
https://github.com/mathesar-foundation/mathesar/blob/a5466472b1dcf10ca0ed06999d555b1db593ae56/mathesar/utils/explorations.py#L198

I think adding the database_id fields to these might resolve this. If you'd like me to submit a PR for this, I'm happy to do so. Otherwise, I hope that this helped identify the fix! Please let me know if you need anything else from me.

```diff
# export.py
- exploration_chunk_gen = exploration_chunker(conn, exploration_id, **kwargs)
+ exploration_chunk_gen = exploration_chunker(conn, exploration_id, database_id=database_id, **kwargs)
columns = next(exploration_chunk_gen)
```

```diff
# explorations.py
def exploration_chunker(
    conn,
+   database_id,
    exploration_id,
    limit=None,
    offset=None,
    batch_size=2000
):
    with connect(database_id, user) as conn:
          # ...
-         exp_model = get_exploration(exploration_id)
+         exp_model = get_exploration(exploration_id, database_id)
```


## Expected behavior
<!-- A clear and concise description of what you expected to happen. -->
The data exploration being exported as CSV and successfully downloaded to the device. 

## To Reproduce
<!-- How can we recreate this bug? Please try to provide a Minimal, Complete, and Verifiable (http://stackoverflow.com/help/mcve) example if code-related. -->
1. Create a database with one of the example schemas (ex. Bike Shop) or use existing database with data
2. Open the Schema with data
3. Select "New Exploration"
4. Select a table with data
5. Select column(s)
6. Save the exploration
7. Select "Export"
8. Observe the error in the container logs and browser download error

## Environment
 - Mathesar version: 0.10.1
 - OS: macOS 26.4.1
 - Browser: Google Chrome
 - Browser Version: 147.0.7727.138 (Official Build) (arm64)
 - Other info: 

## Additional context

The following image is what is shown in the browser when exporting:
<img width="668" height="330" alt="download error in google chrome" src="https://github.com/user-attachments/assets/bc61c2aa-a2d2-4043-a2c3-f61894d52398" />

--- a/mathesar/utils/explorations.py
+++ b/mathesar/utils/explorations.py
@@ -188,6 +188,7 @@ def run_exploration(exploration_def, conn, limit=100, offset=0):
 
 def exploration_chunker(
     conn,
+    database_id,
     exploration_id,
     limit=None,
     offset=None,
@@ -195,7 +196,7 @@ def exploration_chunker(
 ):
     limit = min(limit or 50000, 50000)  # We cap limit at 50000
     # so that we can avoid loading explorations > 50000 rows into memory.
-    exp_model = get_exploration(exploration_id)
+    exp_model = get_exploration(exploration_id, database_id)
     exploration_def = {
         "database_id": exp_model.database.id,
         "base_table_oid": exp_model.base_table_oid,

--- a/mathesar/views/export.py
+++ b/mathesar/views/export.py
@@ -34,7 +34,7 @@ def export_exploration_csv_in_chunks(
 ):
     with connect(database_id, user) as conn:
         csv_buffer = StringIO()
-        exploration_chunk_gen = exploration_chunker(conn, exploration_id, **kwargs)
+        exploration_chunk_gen = exploration_chunker(conn, database_id, exploration_id, **kwargs)
         columns = next(exploration_chunk_gen)
         csv_writer = csv.DictWriter(csv_buffer, fieldnames=columns)
 

